### DISCUSSION TITLE
v5折线图是否可以标记线中的某一段

### DISCUSSION THREAD

**Discussion by explorerEzreal** [antvis/G2#6263] [Q&A]
### 问题描述

在折线图中的某一条线，我需要让处于一定范围内的线，被标记上一个颜色

### 预期行为

参考echarts

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

### 屏幕截图或视频（可选）

<img width="1510" alt="image" src="https://github.com/antvis/G2/assets/84378981/28cfdcfc-c441-4374-8de5-6c645e21b12f">

**Comment by pearmini** [comment]
可以参考一下这个例子：https://g2.antv.antgroup.com/examples/general/line/#line-variable-color
